static/js/components/VideoPlayer.js

```diff
@@ -269,6 +294,9 @@ export default class VideoPlayer extends React.Component<*, void> {
             createEventHandler(event, video.key)
           })
         })
+        if (typeof this.tech_.hls !== "undefined") {
```

Comment:
It is valid to just say `this.tech_.hls !== undefined`. Since ES5 you cannot redefine `undefined`

Reply:
✅